app/com/yetu/youtube/views/index.scala.html

```diff
@@ -18,6 +18,8 @@
     <link rel="stylesheet" href="assets/youtube_producer/styles/app.css"/>
     <script>
         var token = "@devToken";
+        var AUTH_SERVER = "@authUrl";
```

Comment:
I would put all 3 things in a single global variable, to reduce the potential of name clashes